lib/Screens/Contact.dart

```diff
@@ -1,10 +1,144 @@
+import 'dart:async';
+
 import 'package:flutter/material.dart';
+import 'package:url_launcher/url_launcher.dart';
+
+class ContactUs extends StatefulWidget {
+  @override
+  _ContactUsState createState() => _ContactUsState();
+}
 
-class ContactUs extends StatelessWidget {
+class _ContactUsState extends State<ContactUs> {
   @override
   Widget build(BuildContext context) {
     return Scaffold(
+      appBar: AppBar(
+        iconTheme: IconThemeData(
+          color: Colors.pink,
+        ),
+        backgroundColor: Colors.white,
+        elevation: 1,
+        title: Text(
+          "Contact Us",
+          style: TextStyle(color: Colors.black, fontSize: 20),
+        ),
+      ),
+      body: Container(
+        color: Colors.pink[50],
+        child: Column(
+          children: [
+            Container(
+              padding: EdgeInsets.symmetric(vertical: 25, horizontal: 15),
+              child: Card(
+                elevation: 1,
+                shadowColor: Colors.grey,
+                child: Padding(
+                  padding: EdgeInsets.all(30),
+                  child: Text(
+                    'Do you have an issue, or do you want a new functionality, or are you looking for some random answers? Contact us here -',
+                    textAlign: TextAlign.center,
+                    style: TextStyle(fontSize: 20, color: Colors.pinkAccent),
+                  ),
+                ),
+              ),
+            ),
+            Flexible(
+              flex: 2,
+              child: Column(
+                mainAxisAlignment: MainAxisAlignment.start,
+                crossAxisAlignment: CrossAxisAlignment.center,
+                children: [
+                  ClickableCard(
+                    textContent: 'Fork our github repository',
+                    color: Colors.green,
+                    onTap: () async {
+                      await _launchURL(
+                          'https://github.com/Saransh-cpp/BookRentApp-Chapter3');
+                    },
+                  ),
+                  ClickableCard(
+                    textContent: 'Suggest some feature',
+                    color: Colors.purple[300],
+                    onTap: () async {
+                      await _launchURL(
+                          'https://github.com/Saransh-cpp/BookRentApp-Chapter3/issues');
+                    },
+                  ),
+                  ClickableCard(
+                    textContent: 'Contact owner on Github',
```

Comment:
```suggestion
                    textContent: 'Contact author on Github',
```

I don't own all of the code written in here :)